rstest/src/lib.rs

```diff
@@ -1182,6 +1182,30 @@ pub use rstest_macros::fixture;
 /// }
 /// ```
 ///
+/// ## Ignoring Arguments
+///
+/// Sometimes, you may want to inject and use fixtures not managed by rstest
```

Comment:
Just a note: please can you add a little note in _Inject Test Attribute_ with a link to this section?

Reply:
added 👍